onebusaway-android/build.gradle

```diff
@@ -20,7 +20,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:2.0.0'
```

Comment:
Could you remove the modification to this file from this commit?

I believe these are from Android Studio 2.0 - I'd like to bump them in a separate commit.